strawberry/schema/config.py

```diff
@@ -21,5 +24,8 @@ def __post_init__(
         if auto_camel_case is not None:
             self.name_converter.auto_camel_case = auto_camel_case
 
+        if not issubclass(self.info_class, Info):  # pragma: no cover
+            raise TypeError("`info_class` must be a subclass of strawberry.types.Info")
+
```

Comment:
Adding a test for this wouldn't hurt :) Looks like we don't have a fitting test file for it yet (I'd suggest creating `tests/schema/test_config.py`).

Reply:
Cool cool, will tackle this tomorrow.